i18n/fr/docusaurus-plugin-content-docs/version-22.10/monitoring/anomaly-detection.md

```diff
@@ -1,61 +1,63 @@
+que la normale et déclenche une alerte. Cela permet à l'entreprise de se rendre compte d'un piratage.
+
+### Fonctionnement
 
-Les données collectées sont envoyées à la plateforme centrale Centreon Cloud
-afin de pouvoir calculer un modèle de comportement régulier grâce à l'historique
+1. Les données collectées sont envoyées à la plateforme SaaS de Centreon.
+
+2. Centreon calcule un modèle de comportement régulier grâce à l'historique
 de ces données.
 
-Une fois le modèle calculé, les prédictions sont ensuite générées et récupérées
-sur la plateforme Centreon sur site.
+3. Une fois le modèle calculé, les prédictions sont générées et récupérées
+sur votre plateforme Centreon.
 
-Ces prévisions serviront de seuils flottants qui seront ensuite utilisés par le
-moteur de surveillance pour comparer la valeur collectée avec les seuils prévus
-pour mettre en évidence les écarts et générer des alertes.
+4. Les prévisions servent de seuils flottants qui sont ensuite utilisés par le
+moteur de supervision pour comparer la valeur collectée avec les seuils prévus
+pour mettre en évidence les écarts et générer des alertes. Pour ne pas fausser l'analyse, les données reçues pendant les plages de maintenance sont exclues du calcul des prévisions.
 
-![image](../assets/monitoring/anomaly/centreon_cloud.png)
+5. Les modèles sont recalculés régulièrement, et les prévisions s'affinent dans le temps.
 
 ## Prérequis
 
-Le module **Centreon Anomaly Detection** requiert les prérequis suivants :
+Pour utiliser **Anomaly Detection**, vous aurez besoin des éléments suivants :
 
-  - Centreon en version minimale 20.04
-  - Un jeton fourni par Centreon pour accéder à la plateforme Centreon Cloud
-  - Une connexion Internet depuis le serveur Centreon Central
-  - La variable d'environnement SHELL [LC_ALL](https://www.gnu.org/software/gettext/manual/html_node/Locale-Environment-Variables) ne doit pas être définie ou bien avoir la valeur `C`. Pour vérifier la valeur de cette variable, entrez :
+- Un jeton fourni par Centreon pour accéder à la plateforme SaaS de Centreon,
+- Une connexion internet depuis le serveur Centreon central.
+- La variable d'environnement SHELL [LC_ALL](https://www.gnu.org/software/gettext/manual/html_node/Locale-Environment-Variables) ne doit pas être définie ou bien avoir la valeur `C`. Pour vérifier la valeur de cette variable, entrez :
 
-      ```
-      echo $LC_ALL
-      ```
-      
-  - La prédiction fonctionne mieux avec des services surveillés qui présentent
-    un comportement saisonnier comme indiqué ci-dessous :
-
-![image](../assets/monitoring/anomaly/simple_scheme.png)
+  ```
+  echo $LC_ALL
+  ```
 
 ## Installation
 
-### Installation des paquets
+### Étape 1 : Installation des paquets
 
-Ajouter un référentiel supplémentaire:
+1. Ajoutez un dépôt supplémentaire:
 
 <Tabs groupId="sync">
 <TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
 
-Rien à faire
```

Comment:
The "Rien à faire" and the command for Centos 7 are shifted to the left from the content of point 2

Reply:
This is because of the set of tabs. That's how Docusaurus works. It's very ugly but I can't do anything about it, I'm afraid.